apps/app/components/Register/index.tsx

```diff
@@ -122,14 +122,14 @@ function Register({ cities }: any) {
               rules={[
                 {
                   required: true,
-                  pattern: /^[0-9]{9}$/,
+                  pattern: /^\+351[0-9]{9}$/,
```

Comment:
This is the pattern used by bokken 🙏🏼

```suggestion
                  pattern: /^\+351[12356]{9}$/,
```